Xamarin.Forms.Platform.WP8/ButtonRenderer.cs

```diff
@@ -20,7 +21,9 @@ protected override void OnElementChanged(ElementChangedEventArgs<Button> e)
 			base.OnElementChanged(e);
 
 			var button = new WButton();
+			button.ClickMode = ClickMode.Press;
 			button.Click += HandleButtonClick;
+			button.AddHandler(UIElement.TapEvent, new EventHandler<GestureEventArgs>(HandleButtonTap), true);
```

Comment:
Yes, I think so, Just like we didn't remove Click event handler here. (e,g, button.Click -= HandleButtonClick)